### ISSUE TITLE
Remove GARD

### ISSUE THREAD

**Issue by joeflack4** [monarch-initiative/mondo-ingest#399]
## Overview
This is causing errors when Joe runs `build-mondo-ingest`. There are several places ot remove.

Joe has many more notes in Workflowy.

**Comment by matentzn** [comment]
Did you use my branch? There were quite a few run errors like GARD, also do. I am systematically eliminating them atm, most of them are rooted in the SSSOM py migration to curies converters. In any case, don't worry now, I will get this to work.

**Comment by joeflack4** [comment]
> Did you use my branch?
No. I was checking on the results of `build-mondo-ingest` on my branch on Tuesday. 

> don't worry now, I will get this to work.
I won't, but I feel bad man, that you have to be the one to do this, especially what with how you're trying to allocate your time now!

Trish and I chatted briefly again on how to run `build-mondo-ingest` in a modular way that can pick up where it left off. I don't know when we can prioritize that, but it would be nice.

**Comment by joeflack4** [comment]
@matentzn Just want to confirm. Sorry I haven't gotten to this yet; I'm still good to remove this, yes?

**Comment by matentzn** [comment]
Except what is in externally managed content! From the regular pipeline, yea
